src/sass/static/affiliates_ib.scss

```diff
@@ -27,30 +27,222 @@
+        min-height: 417px;
+
+        &-header {
+            font-weight: 300;
+            line-height: 45px;
+            font-size: 36px;
+            color: $COLOR_WHITE;
+        }
     }
-    .faq-box {
-        padding: 1rem;
+    .statistics {
+        display: flex;
+        align-items: center;
+        background: url('../images/pages/contact/banner-bg.svg');
+        min-height: 200px;
+
+        h3 {
+            margin: 0;
+        }
     }
-    .table-container.center-text {
-        & th, td {
-            text-align: center;
+    .how-it-works {
+        background: url('../images/pages/affiliates_ib_landing/background/orange.png');
+        background-size: cover;
+        color: $COLOR_WHITE;
+
+        h2, h3 {
+            color: $COLOR_WHITE;
+            margin: 20px 0;
+        }
+        h3 {
+            font-weight: 700;
+        }
+        p {
+            font-size: 22px;
+            font-weight: normal;
+            line-height: 28px;
         }
     }
-    @media screen and (max-width: 480px) {
-        table, .mobile-text-small {
-            font-size: 0.9rem;
+    .who-apply {
+        display: flex;
+        align-items: center;
+
+        .gr-row {
+            max-width: 780px;
+        }
+        h2 {
+            margin: 20px 0;
+        }
+        h4 {
+            font-weight: 700;
+        }
+        button {
+            margin: 20px 0;
+        }
+    }
+    .why-partner {
+        min-height: 580px;
+        display: flex;
+        align-items: center;
+
+        h2 {
+            margin-top: 25px;
+        }
+        h4 {
+            margin-top: 24px;
+            margin-bottom: 15px;
+        }
+        img {
+            height: 64px;
+            width: 64px;
+        }
+        @media screen and (max-width: 480px) {
+            h4 {
+                font-size: 1rem;
+            }
+            img {
+                height: 56px;
+                width: 56px;
+            }
+        }
+    }
+    .payment-method {
+        .container {
+            max-width: 1012px;
+        }
+        .gr-row {
+            max-width: 940px;
+            margin: 0 auto;
+        }
+        h2 {
+            margin: 25px 0;
         }
     }
-}
\ No newline at end of file
+    .type-of-partner {
+        .container {
+            max-width: 780px;
+
+            & > p {
+                font-weight: 300;
+            }
+            h2 {
+                margin: 20px 0;
+            }
+            h3 {
+                margin: 20px 0;
+            }
+            .has-tabs {
+                margin: 20px 0;
+
+                .ui-tabs-nav {
+                    border-width: 2px;
+                    border-color: $COLOR_ORANGE;
+                    margin-bottom: 0;
+
+                    li {
+                        padding: 10px;
+                        margin: 0;
+                        white-space: normal;
+
+                        a {
+                            width: 100%;
```

Comment:
could you add `outline: none` here?

Reply:
@kellybinary Can you tell me why add that?